source/_integrations/syncthru.markdown

```diff
@@ -13,30 +14,12 @@ ha_domain: syncthru
 The Samsung SyncThru Printer platform allows you to read current data from your local Samsung printer.
 
 It usually provides information about the device's state, the left amount of ink or toner and the state of paper trays.
-The platform automatically monitors every supported part.
-
-If you wish not to include certain monitored values specify the values that you would like to see in the front-end via the `monitored_conditions` setting.
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: syncthru
-    resource: http://my-printer.address
-    name: My Awesome Printer
-```
-
-{% configuration %}
-  resource:
-    description: The address for connecting to the printer. Equal to the SyncThru Webservice address.
-    required: true
-    default: false
-    type: string
-  name:
-    description: A user specified name for the printer. Defaults to "Samsung Printer" and the friendly name will be the name of the printer model.
-    required: false
-    default: Samsung Printer
-    type: string
-{% endconfiguration %}
+
+### Setup
+
+Go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Samsung SyncThru Printer**.
+
+If your printer is on and you have enabled [SSDP](/../ssdp) discovery, it's likely that the printer is automatically discovered. In that case, you shouldn't need any information to complete configuration - simply confirm that the information displayed is correct.
```

Comment:
```suggestion
If your printer is on and you have enabled [SSDP](/integrations/ssdp) discovery, it's likely that the printer is automatically discovered. In that case, you shouldn't need any information to complete configuration - simply confirm that the information displayed is correct.
```

Reply:
Good catch, thanks. I opted to change it to `../ssdp/` instead, as hardcoding `integrations` does not seem to serve a purpose here.

Reply:
Actually, we should not do that @scop. I'm committing the suggestion. Using a standard is more important if we ever change the structure.

Reply:
@frenck if you prefer it that way:
```
$ git grep -F ../ source/_integrations/
source/_integrations/huawei_lte.markdown:or using YAML. Additionally, if the [SSDP integration](../ssdp/) is
source/_integrations/ssdp.markdown: - [deCONZ](../deconz/)
source/_integrations/ssdp.markdown: - [Huawei LTE](../huawei_lte/)
source/_integrations/ssdp.markdown: - [Philips Hue](../hue/)
source/_integrations/ssdp.markdown: - [Samsung TV](../samsungtv/)
source/_integrations/ssdp.markdown: - [UPnP](../upnp/)
source/_integrations/ssdp.markdown: - [Sony Songpal](../songpal/)
source/_integrations/vallox.markdown:For convenient switching of ventilation profiles in the GUI, consider using an [input_select](../input_select) hooked to an automation, for example:
```